cloudburst/server/scheduler/policy/default_policy.py

```diff
@@ -292,9 +316,9 @@ def pin_function(self, dag_name, function_ref, colocated):
     def commit_dag(self, dag_name):
         for function_name, location in self.pending_dags[dag_name]:
             if function_name not in self.function_locations:
-                self.function_locations[function_name] = set()
+                self.function_locations[function_name] = []
```

Comment:
why are we changing from set to list?

Reply:
This allows us to do round-robin scheduling. `set` don't have order but `list`s do.

Reply:
got it.